web_responsive_company/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020 - Today: GRAP (http://www.grap.coop)
```

Comment:
Change to 2021 :smile:

Reply:
it's an update of previous module I developped ``web_favorite_company`` in 2021.
https://github.com/OCA/web/pull/1688
(but after few monthes of use in my company, it requires extra settings, so I prefer a more simple "search functionnality")

but no problem, I'll change it.

Reply:
Ok.. ok perfect!!

Reply:
done.

Reply:
done.